mlmodel/format/ItemSimilarityRecommender.proto

```diff
@@ -3,25 +3,6 @@
 // Use of this source code is governed by a BSD-3-clause license that can be
 // found in LICENSE.txt or at https://opensource.org/licenses/BSD-3-Clause
 
-/*
```

Comment:
Is this deletion deliberate?

Reply:
Yes, this deletion is deliberate. This comment doesn't belong here. It was somehow copied from `TreeEnsemble.proto`.